### DISCUSSION TITLE
Placeholder showed after focus out of selectbox when clear button clicked

### DISCUSSION THREAD

**Discussion by martinbarilik** [orchidjs/tom-select#97] [Q&A] [answered]
Sorry for bombing you with all the bugs, but i really want to use tomselect :D 

https://jsfiddle.net/mArtinko5MB/5genrL7w/1/

When you clear the select, placeholder is shown only when clicking elsewhere ( on focus off ), is this intentional ?

**Comment by oyejorge** [comment, marked as answer]
> Sorry for bombing you with all the bugs, but i really want to use tomselect

No worries, you're finding real problems and giving me good jsfiddle examples so I don't have to dig around for the problem.

> When you clear the select, placeholder is shown only when clicking elsewhere ( on focus off ), is this intentional ?

That was not intentional, [fixed with this commit](https://github.com/orchidjs/tom-select/commit/628e3556726acac1129786fb8611feafcc34e757)
